qemu/tests/secure_execution.py

```diff
@@ -0,0 +1,38 @@
+from virttest import error_context
+from virttest import virt_vm
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Secured guest boot test:
+    1) Log into a secure guest
+    2) Check if there's error messagein
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    error_context.context("Try to log into guest '%s'." % vm.name,
+                          test.log.info)
+    if params.get("start_vm") == "no" and not params.get_boolean("error_check"):
```

Comment:
Seems `start_vm` is always to `no` in this case.

Reply:
this is for future cases in this cfg

Reply:
updated